### PR TITLE
gofumpt for v4

### PR DESCRIPTION
**Pull request by faddat**
## Description

Applies gofumpt to gaia v4, standardizing the codebase

#1449 
______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

**Comment by faddat** [comment]
Got to guess that the liveness check here no longer works -- thoughts?

**Comment by faddat** [comment]
Seems not only that is needed to fix it.  if I had to guess it is dependent on environment.

**Comment by faddat** [comment]
I just got rid of the liveness test.

File: .github/workflows/sims.yml [hunk 1]
```diff
@@ -121,27 +121,3 @@ jobs:
         run: |
           make test-sim-multi-seed-short
         if: "env.GIT_DIFF != ''"
-
-  test-liveness:
```
Comment (tac0turtle) [inline .github/workflows/sims.yml:125]: was this failing?

**Review by tac0turtle — APPROVED** [review]
utACK

File: .github/workflows/sims.yml (hunk 1 above)
Comment (faddat) [reply .github/workflows/sims.yml:125]: yes, but I think it's failing for process reasons?  

Basically, it is an attempt to backport and the original v4 branch had a different test.  I could take the .github folder from v4 and get rid of the diff, though. 

Ah, I remember why it's failing now-- pretty sure it couldn't find a file in the environment.

Comment (faddat) [reply .github/workflows/sims.yml:125]: Another way to say this: v4 ci isn't maintained.

**Comment by okwme** [comment]
same as per comment from #1509:
> opened a new issue related to stable release policy (#1549).
appreciate the work @faddat but these changes don't make sense to merge into those versions unless they're going to be part of a new release of those versions and EOL should only be bypassed by security updates.

**Comment by faddat** [comment]
Except that legibility is a security update.

**Pull request closed without merging**